### ISSUE TITLE
Can't scroll back when entering email/password values too long

### ISSUE THREAD

**Issue by sagehane** [bitwarden/android#4209]
### Steps To Reproduce

1. Type a sufficiently long email or password into their respective fields
2. Observe that the visible portion of the field shifts such that the beginning characters cannot be seen

### Expected Result

The prompt should support scrolling back to see what was previously typed.

### Actual Result

The prompt doesn't allow scrolling back. This is a regression in feature compared to the previous app and makes it harder for me to see if I typed my master password correctly.

### Additional Context

Might also be considered a feature request depending on how one looks at it?

I installed v2024.10.2 via [Obtanium](https://github.com/ImranR98/Obtainium) and reproduced the following with `com.x8bit.bitwarden.apk` and `com.x8bit.bitwarden.beta.apk`.

### Build Version

v2024.10.2

### What server are you connecting to?

N/A

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by aj-rosado** [comment]
Hello,

I cannot reproduce it, could you please retest it on the most recent version (2024.11.6) and provide more information like the screen it happens, the OS and the phone model?

Thank you!

**Comment by aj-rosado** [comment]
@sagehane

**Comment by sagehane** [comment]
Sorry, I just noticed that I was being dumb and using the app wrong. I can indeed scroll if I just swipe on the textbox itself; I just didn't notice as I don't think I ever encountered this in other apps I've used.

A better step to reproduce what I was experiencing (Tested on version 2024.11.6):
1. Type something long enough into any textbox such as the initial login, password prompt, fields for entries, etc.
2. Click on a character in the box and wait a blue bubble to show.
3. Hold that bubble and move it to either side to try to "scroll" to the previous text.
4. Fail as I can only move the bubble until I hit the end of the visible portion of the screen.

I thought the above was the only way to do such a thing as I was used to it from trying to scroll on the URL bar or apps like Firefox. I guess I'll close this issue as "not planned" as it wasn't an issue to begin with.
